docs/admin/scim/oauth-protection.md

```diff
@@ -2,16 +2,24 @@
 tags:
   - administration
   - scim
+  - oauth-protection
 ---
 
```

Comment:
Consider writing a couple of lines that explain what will be achieved by the steps below.